IdentityCore/src/webview/systemWebview/session/MSIDASWebAuthenticationSessionHandler.m

```diff
@@ -82,10 +82,12 @@ - (void)startWithCompletionHandler:(MSIDWebUICompletionHandler)completionHandler
         {
             NSError *cancelledError = MSIDCreateError(MSIDErrorDomain, MSIDErrorUserCancel, @"User cancelled the authorization session.", nil, nil, nil, nil, nil, YES);
             
+            self.webAuthSession = nil;
             if (completionHandler) completionHandler(nil, cancelledError);
             return;
         }
         
+        self.webAuthSession = nil;
```

Comment:
Do we want to nil out `webAuthSession` before line 76 too?

Reply:
Good point. We should. Addressed